pkg/hwinfo/hwinfo_nonwindows.go

```diff
@@ -17,6 +19,57 @@ import (
 	"github.com/cloudradar-monitoring/cagent/pkg/common"
 )
 
+type cpuInfo struct {
+	manufacturer      string
+	manufacturingInfo string
+	description       string
+	coreCount         string
+	coreEnabled       string
+	threadCount       string
+}
+
+var (
+	Timeout    = 3 * time.Second
+	ErrTimeout = errors.New("invoker: command timed out")
+)
+
+type Invoker interface {
```

Comment:
Is `Invoker` only useful for `hwinfo`? If not does it make sense to put it in the `common` package or its own one?
Also add a comment what is `Invoker` to be used for

Reply:
done